stories/bar/bar.stories.js

```diff
@@ -462,6 +495,7 @@ export const barAsFloatingFooter = () => `
 
 barAsFloatingFooter.parameters = {
     docs: {
-        iframeHeight: 350
+        iframeHeight: 150,
+        storyDescription: 'The floating footer bar should also display only finalizing action buttons that impact the entire page. This type floats above the page at the bottom of the screen and is always visible when scrolling. It can be displayed by adding the <code>--floating-footer</code> modifier class to the container element.'
```

Comment:
@InnaAtanasova I imagine this also needs to be removed right? _"This type floats above the page at the bottom of the screen and is always visible when scrolling."_

Reply:
(Spoke with Inna on Slack, and she suggested the line stays.)